Fix superscripts in OSIS->xhtml translations, based on a fix
by Karl Kleinpaste (revision 3375 in SVN)

diff --git a/src/modules/filters/osisxhtml.cpp b/src/modules/filters/osisxhtml.cpp
index fb743e2..759acc4 100644
--- a/src/modules/filters/osisxhtml.cpp
+++ b/src/modules/filters/osisxhtml.cpp
@@ -617,7 +617,7 @@ bool OSISXHTML::handleToken(SWBuf &buf, const char *token, BasicFilterUserData *
 				else if (type == "ol") {
 					outText("</span>", buf, u);
 				}
-				else if (type == "sup") {
+				else if (type == "super") {
 					outText("</sup>", buf, u);
 				}
 				else if (type == "sub") {
